parse/hex.js

```diff
@@ -1,4 +1,9 @@
 function hex(hex) {
+  if (hex.length === 4) {
+    hex = '#' + hex[1] + hex[1] +
+      hex[2] + hex[2] + 
+      hex[3] + hex[3];
+  }
```

Comment:
Could you switch to using `charAt` instead of index notation? http://stackoverflow.com/questions/5943726/string-charatx-or-stringx